.github/workflows/sonarcloud.yml

```
@@ -0,0 +1,44 @@
+name: SonarCloud
+on:
+  push:
+    branches: [ main ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)